crates/services/txpool_v2/src/collision_manager/mod.rs

```diff
@@ -49,27 +49,26 @@ impl<Idx> Collisions<Idx> {
     }
 }
 
-pub trait CollisionManagerStorage {
+pub trait CollisionManager {
+    /// Storage type of the collision manager.
+    type Storage;
+    /// Index that identifies a transaction in the storage.
     type StorageIndex: Copy + Debug;
```

Comment:
Is the `Copy + Debug` an over-constraint here?

Reply:
Removed.